org.sil.scripture-app-builder.yml

```diff
@@ -0,0 +1,114 @@
+# REFERENCES:
+  # Python2.7:  https://github.com/flathub/shared-modules/blob/cc50bdbb34cc3d29bf61cfea5e27cd980b7bda04/python2.7/python-2.7.json
+  # OpenJDK17: https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk17/blob/branch/23.08/org.freedesktop.Sdk.Extension.openjdk17.yaml
+  # Node20:    https://github.com/flathub/org.freedesktop.Sdk.Extension.node20/blob/branch/23.08/org.freedesktop.Sdk.Extension.node20.yaml
+# FIRST-TIME-BUILD: ~30m on 10-20Mb/s download speed
+app-id: org.sil.scripture-app-builder
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+  # - org.freedesktop.Sdk.Extension.openjdk21
+finish-args:
+  # Window x11 socket access
+  - --socket=x11
+  # Audio access
+  - --socket=pulseaudio
+  # Unsure if the program absolutely needs ssh
+  # - --socket=ssh-auth
+  # Or the printer socket
+  # - --socket=cups
+  # Network access
+  - --share=network
+  # XShm access
+  - --share=ipc
+  # All external devices (catch all for devices we interact with and graphical acceleration)
+  - --device=all
+  # Local file browsing access
+  - --filesystem=host
+  # Temporary file access
+  - --filesystem=/tmp
+  # Filesystem mounts
+  - --filesystem=/mnt
+  # USB Drives
+  - --system-talk-name=org.freedesktop.Udisks2
+  # File saving access
+  - --system-talk-name=org.freedesktop.FileManager1
+
+# Build args give resource access during the build
+build-options:
+  build-args:
+    - --share=network
+
```

Comment:
Network access during build is not allowed on Flathub

Reply:
We download a docker image of the pre-built binaries from a private repo. Is this flag not necessary to do that?

Reply:
This is clearly mentioned in the requirements page linked in the PR checklist.

You will have to find an alternate building mechanism

Reply:
If you want to use prebuilt binaries, you can host them somewhere and directly use those in the source. Not the docker image.

Reply:
We will be moving the prebuilt binaries to an S3 Bucket and access through url and sha256

Reply:
Removed build-args for `--share=network` and using zip file from S3 bucket.